termux-shared/src/main/java/com/termux/shared/termux/TermuxConstants.java

```diff
@@ -1046,6 +1049,9 @@ public static final class TERMUX_SERVICE {
              * be created in {@link #EXTRA_RESULT_DIRECTORY} if {@link #EXTRA_RESULT_SINGLE_FILE} is
              * {@code false} for the TERMUX_SERVICE.ACTION_SERVICE_EXECUTE intent */
             public static final String EXTRA_RESULT_FILES_SUFFIX = TERMUX_PACKAGE_NAME + ".execute.result_files_suffix"; // Default: "com.termux.execute.result_files_suffix"
+            /** Intent {@code long} extra for graceperiod between SIGTERM and SIGKILL
+             * for the TERMUX_SERVICE.ACTION_SERVICE_STOP intent */
+            public static final String EXTRA_TERMINATE_GRACE_PERIOD = TERMUX_PACKAGE_NAME + ".execute.stop.delay"; // Default: "com.termux.execute.stop.delay"
```

Comment:
This should be `stop_delay`

Reply:
Applied

Reply:
The extra name was  also different. Both should be same, naming them `sigkill_delay_on_stop` seems better, with respective case.

Reply:
Will have to rename `gracePeriod` and `gracePeriodMsec` as well